helpers/config.js

```diff
@@ -110,6 +110,10 @@ function Config(packageJson) {
 		appConfig.coverage = true;
 	}
 
+	if (appConfig.wsEngine === undefined || appConfig.wsEngine === null) {
+		appConfig.wsEngine = 'sc-uws';
+	}
+
```

Comment:
I would prefer to have this option `wsEngine` under `peers.options` configuration. If you do so, please move `wsWorkers` option as well under `peers.options`